Example/test/index.js

```diff
@@ -92,6 +94,14 @@ export default class Example extends Component {
     // const transX = new Value(0);
     const clock = new Clock();
     // const twenty = new Value(20);
+    const a = param("a");
+    const b = param("b");
+    const calc = proc(add(multiply(a, b), 333), a, b);
```

Comment:
Can you add some better example?

Reply:
Yes I have another example where I have converted all easings to use `proc` nodes - should I commit that as well?